src/react/databrowser/objects/ObjectDelete.tsx

```diff
@@ -137,29 +209,72 @@ const ObjectDelete = ({
       isOpen={true}
       title="Confirmation"
     >
-      <div> {title(toggled, bucketInfo.isVersioning)} </div>
+      <div> {title(toggledFiles, bucketInfo.isVersioning)} </div>
       <Files>
         <Table>
           <T.Body>
-            {toggled.map((s) => (
+            {toggledFiles.map((s) => (
               <T.Row key={s.key}>
                 <T.Cell>
                   {' '}
                   {s.key}
                   <VersionId hidden={!s.versionId}> {s.versionId} </VersionId>
                   <Description>
                     {s.size && <PrettyBytes bytes={s.size} />}
+                    {(s.lockStatus === "LOCKED") && <div
+                      style={{
+                        color: theme.brand?.textTertiary,
+                      }}
+                    >
+                      <Icon style={{ display: 'inline', marginRight: '0.57rem' }} className="fa fa-lock"></Icon>
+                      {confirmed ? <span>Protected, will be deleted</span> : <span>Protected, will not be deleted</span>}
+                    </div>}
                   </Description>
                 </T.Cell>
+                <RemoveCell>
+                  {' '}
+                  <div onClick={() => removeFile(s.key)}>
+                    {' '}
+                    <i className="fas fa-times"></i>{' '}
+                  </div>{' '}
+                </RemoveCell>
               </T.Row>
             ))}
           </T.Body>
         </Table>
       </Files>
-      <div>
+      <div style={{ marginBottom: '1.25rem' }}>
         {' '}
         Total: <PrettyBytes bytes={totalSize} />{' '}
       </div>
+      {hasLockedFiles && <Fragment>
+      <SpacedBox
+        mr={12}
+        style={{
+          margin: '1rem',
+        }}
```

Comment:
```suggestion
      <SpacedBox
        mr={12}
        ml={}
        mt={}
        mb={}
```

For me, the purpose of SpaceBox is to define the margin. It doesn't make sense for me to add another style with margin. You can add `ml`, `mt` and `mb` if you want ;)